HHS/OPRE-OPS
# feat: adds CAN Funding Received form

## What changed

- adds CAN Funding Received Form
- adds validation
- handles cancel actions
- handles api calls for both forms
- updates line graph logic
- adds `placeholder` prop for CurrencyInput component allowing for placeholder values to be added

## Issue

#310 #3100

## How to test

1. watch e2e tests pass or manually:
2. login as budget team
3. goto `/cans/504/funding`
4. click on edit button
5. enter 5,000,000 for CAN Budget
6. click on Add FY Budget
7. add 1,000,000 for Funding Received
8. click on Add Funding Received Button
9. Verify amount is shown in Funding Received YTD Card
10. Verify that amount is added to the Funding Received YTD Table
11. Click on Save changes
12. Verify Funding Received additions are persisted

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
